pkg/services/serviceaccounts/serviceaccounts.go

```diff
@@ -0,0 +1,9 @@
+package serviceaccounts
+
+import (
+	"context"
+)
+
+type Service interface {
+	DeleteServiceAccount(ctx context.Context, orgID, serviceAccountID int64) error
```

Comment:
```suggestion
```

Reply:
We don't need to export this, there are no users.

Reply:
--
we make use of the `api` namespace/package in the api registeration.
This means we cannot create the function in the `serviceaccounts` package